src/main/webapp/app/exercise/synchronization/services/code-editor-file-sync.service.ts

```
@@ -0,0 +1,761 @@
+    reset(): void {
+        this.incomingMessageSubscription?.unsubscribe();
+        this.incomingMessageSubscription = undefined;
+        this.fileDocs.forEach((entry) => {
+            if (entry.pendingInitialSync?.timeoutId) {
+                clearTimeout(entry.pendingInitialSync.timeoutId);
+            }
+            entry.awareness.destroy();
+            entry.doc.destroy();
+        });
+        this.fileDocs.clear();
+        this.renameTimeouts.forEach((timeoutId) => clearTimeout(timeoutId));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)